my_list.rb

```diff
@@ -0,0 +1,11 @@
+class MyList
+  include Enumerable
+
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] In this activity, we actually expect you to differently create your own `each` method from scratch instead of using the specified inbuilt array method. It should be similar to the example below:

```ruby
class MyList

[...]

def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end

[...]

end
```

Reply:
Hi @rloterh although I appreciate the review, I don’t believe the project specification requires us to implement an each method from scratch and none of my other partners were required to in order to get approved. Please clarify if this is the case or not 😄